### DISCUSSION TITLE
Can I use the keep markup in the highlight() function?

### DISCUSSION THREAD

**Discussion by hoiheart** [PrismLibrary/Prism#2301] [General]
Hello,

The VueJS plug-in I make needs the help of Prism highlight.
But there is a limit to using highlight(raw function), so I ask the question.

```js
Prism.highlight('<div class="hello<vue-diff-modified> hi</vue-diff-modified>"></div>', Prism.languages.html, 'html')
```

Certain tags (vue-diff-modified) want to maintain a markup without conversion (in all Grammar)

1. Can I use the keep markup plug-in?
The current keep markup's hook is not supported in highlight().

2. Is there a way to skip Grammar for a particular pattern of strings?
It seems to me that this is more necessary.

Have a nice weekend.

**Comment by brianlagunas** [comment]
You have posted in the wrong repo. This is for the Prism Library which is an application framework for XAML app development.

**Comment by hoiheart** [comment]
I misunderstood the repo.
I'm sorry. 🙏
